store/tikv/region_cache.go

```diff
@@ -525,6 +534,38 @@ func (c *RegionCache) GetTiKVRPCContext(bo *Backoffer, id RegionVerID, replicaRe
 	}, nil
 }
 
+func (c *RegionCache) GetAllValidTiFlashStores(id RegionVerID, currentStore *Store) []uint64 {
+	allStores := make([]uint64, 0, 4)
```

Comment:
What does capacity `4` mean?

Reply:
I think in most of the case, the tiflash replica should be less than 4, maybe I can use 2 instead.

Reply:
I thought maybe we can add explaination for this assumption.